recipes/swat/meta.yaml

```diff
@@ -0,0 +1,63 @@
+{% set name = 'swat' %}
+{% set version = '1.3.1' %}
+{% set build = '0' %}
+
+package:
+    name: {{ name }}
+    version: {{ version }}
+
+source:
+    url: https://github.com/sassoftware/python-swat/releases/download/v{{ version }}/python-swat-{{ version }}-linux64.tar.gz  # [linux64]
+    sha256: 205084f8e5e10dd582655f796ac579adc2e81b4cfc99007315c48b19127e1e3b  # [linux64]
+    url: https://github.com/sassoftware/python-swat/archive/v{{ version }}.tar.gz  # [not linux64]
+    sha256: bfb2acb228a42e6916747402f932d6fa93fba19bf1939c6c57e2f76532482180  # [not linux64]
+
+build:
+    number: {{ build }}
+    script: python -m pip install --no-deps --ignore-installed .
+    binary_relocation: True
+    binary_has_prefix_files:
+        - lib/linux/
+        - lib/win/
+        - lib/mac/
+
+requirements:
+    build:
+        - python
+        - pip
+        - setuptools
+        - pandas
+        - six
+        - requests
+        - libstdcxx-ng  # [linux64]
```

Comment:
Why are these needed?  It doesn't seem as if the source on github has any c things around

Reply:
I'd recommend that you add ` - compiler('cxx') ` here instead of having libgcc and libstdcxx.  
If this library ships precompiled binaries in its tarball releases this may be problematic when we switch compilers.

Reply:
We do use pre-compiled binaries that are distributed in the tarball.  I don't know that the C dependencies are really needed since these are part of the manylinux1 spec; this package should run on most Linux distributions.  I just added them because conda build suggested it when I was writing the recipe.  I can change it to compiler('cxx') (or maybe just remove them?).

Reply:
I think you can just remove that and also from run, if you really need that please add

```
  - {{ compiler('cxx') }}
  - {{ compiler('c') }}
```

into the ``build`` requirements

Reply:
I think ``setuptools`` will not be necessary as well. Could you please try to remove it?